test/Share.test.js

```diff
@@ -0,0 +1,87 @@
+/*global require,describe,beforeEach,afterEach,it*/
+'use strict';
+
+var expect = require('expect.js');
```

Comment:
Why is this module not imported using es6 imports? I believe it's possible using denodeify or something like that.